internal/pkg/cache/redis/redis.go

```diff
@@ -16,16 +16,19 @@ package redis
 
 import (
 	"context"
-	"github.com/go-redis/redis/extra/redisotel/v8"
-	"github.com/go-redis/redis/v8"
+	"log"
+	"strings"
 	"time"
 
+	"github.com/redis/go-redis/extra/redisotel/v9"
+	"github.com/redis/go-redis/v9"
+
```

Comment:
Nit: remove extra line and sort alphabetically.

Reply:
Fixed.